### PR TITLE
Fixes #1061 Dockerize chat.susi.ai and Auto deploy button for repo

### LINKED ISSUE #1061: Dockerize chat.susi.ai and Auto deploy button for repo
**Actual Behaviour**
Currently, the repo is not installable via Docker.

**Expected Behaviour**

Adding Dockerfile for installation ease and Auto-deployment button.

**Steps to reproduce it**

Add steps to reproduce bugs or add information on the place where the feature should be implemented. Add links to a sample deployment or code.

**LogCat for the issue**

Provide logs for the crash here

**Screenshots of the issue**

Where-ever possible attach a screenshot of the issue.

**Would you like to work on the issue?**

Yes, anna work on this issue.

### PR DESCRIPTION
**Pull request by bhaveshAn**
Fixes #1061 
Changes: [Add here what changes were made in this issue and if possible provide links.]

Added `Dockerfile` and Auto deployment button for the same in `README.md`

Demo Link: https://pr-[ADD_PULL_REQUEST_NUMBER_HERE]-fossasia-susi-web-chat.surge.sh

Screenshots for the change: 

@rishiraj824 @madhavrathi @mariobehling Please review. Thanks!!

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+WORKDIR /usr/src/app
+
+# copy requirements
+COPY package.json /usr/src/app/
+COPY .eslintrc /usr/src/app/
+COPY .angular-cli.json /usr/src/app/
+
+# install requirements
+RUN npm install -g @angular/cli@latest
+RUN npm install
```
Comment (daminisatya) [inline Dockerfile]: Can you write it on a single line? 

```
RUN <command> \
        <command> 
```

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,22 @@
+FROM node:boron-alpine
```
Comment (daminisatya) [inline Dockerfile]: Why boron-alpine?

Comment (bhaveshAn) [reply Dockerfile]: Firstly. I added `boron`, but then changes suggested to use `boron-alpine`

Comment (bhaveshAn) [reply Dockerfile]: So, finally what should I add `boron` or `boron-alpine` ??

Comment (daminisatya) [reply Dockerfile]: I'm fine with boron-alpine, it finally boils down to docker optimization which we can look into it later. @bhaveshAn

**Comment by bhaveshAn** [comment]
@daminisatya Sure :+1: 
Can you please merge it .

File: Dockerfile (hunk 1 above)
Comment (bhaveshAn) [reply Dockerfile]: Can we write it as 
```
RUN npm install -g @angular/cli@latest && \
    npm install
```

**Pull request merged**